src/core/infrastructure/log.js

```diff
@@ -1,4 +1,4 @@
-/*eslint-disable  no-console*/
+/*eslint-disable  no-console, no-unused-vars*/
```

Comment:
Because very often to debug something you comment this console invokations and es-lint doesn't like it